api/endpoint-tests/users/get.js

```diff
@@ -69,7 +69,7 @@ tap.test('users endpoint | GET /users/:userID', async getUserTest => {
             name: null,
             position: null,
             phone: null,
-            state: null
```

Comment:
For some reason, the User model had both `state` and `state_id` fields.  While I was in here, I got rid of the `state` field since it wasn't being used in the relationships anywhere, and I updated the code throughout to use the `state_id` field instead.